### ISSUE TITLE
How to regular the SAM inferred PloygonLabels ???

### ISSUE THREAD

**Issue by SonGohan757** [heartexlabs/label-studio#4597]
I think this is a abnormal result.
https://drive.google.com/file/d/1FbsdfAl4XJKzBq8IRPViThMvChR7h_-v/view?usp=drive_link

**Comment by stone100010** [comment]
哈哈哈哈，笑死我了

**Comment by AymanTalkhani** [comment]
@mi4da @stone100010 If you're on the lookout for AI-made polygon masks, have you tried using [InstaLabel](https://www.instalabel.ai/)? It automates all segmentation/bounding box labeling with custom AI models. All you need to do is upload your dataset, select and label a few samples you'd like to finetune the model on, and hit the 'Train annotator' button. You can then head over to the 'Annotate' tab, click 'Instant Annotation' and have the finetuned model label the rest of your dataset for you. There's even a crowdsourcing option to correct AI predictions for verified ground truth labels. Could be useful for your application.
